src/localization/locales/en.json

```diff
@@ -80,8 +80,21 @@
                     "label":"Local folder",
                     "labelInBrowser":"Local folder (This is only enabled in the desktop version from: https://github.com/Azure/azure-iot-explorer/releases)",
                     "infoText": "Use your local folder as a model repository. The model definition files you wish to use should have a json extension. The first JSON file with an '@id' property matching the digital twin model identifier (DTMI) will be retrieved.",
-                    "textBoxLabel": "Local folder path",
-                    "placeholder": "f:/"
+                    "textBoxLabel": "Selected folder",
+                    "folderPicker": {
+                        "command": {
+                            "openPicker": "Pick a folder",
+                            "navigateToParent": "Navigate back to parent folder",
+                            "select": "Select",
+                            "cancel": "Cancel"
+                        },
+                        "dialog": {
+                            "title": "Select a folder as a local model repository",
+                            "subText": "Selected folder: {{folder}}",
+                            "noFolderFoundText": "No folders found.",
```

Comment:
No folder found

Reply:
plural seems more common. If you try to open folder from vs code, the file explorer would show 'no items found'.